front/src/utils/date.ts

```diff
@@ -162,3 +164,9 @@ export function convertIsoUtcToLocalTime(isoUtcString: string): string {
 export function addDurationToIsoDate(startTime: string, duration: string) {
   return dayjs(startTime).add(ISO8601Duration2sec(duration), 'second').format();
 }
+
+export function transformDateWithoutYear(dateTime: string) {
```

Comment:
> Weirdly, the input is a string and not a Date type.

Means that the function **MUST** take a Date and not a string.

```suggestion
export function serializeDateTimeWithoutYear(dateTime: Date) {
```

> [!NOTE]
> I suggest renaming the function name since this function doesn't transform the given argument but serializes it.